src/envo/envo-edit.owl

```diff
@@ -27839,21 +27861,181 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02500044> ObjectSomeValuesFrom(<
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000014> "cirque basin"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000014> <http://purl.obolibrary.org/obo/ENVO_03000015>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000015> (geographic basin)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000015> "A depression which is part of a planetary crust, is of geographic scale, and is partially or completely enclosed.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000015> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000015> "https://en.wikipedia.org/wiki/Basin")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000015> "geographic basin"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000015> <http://purl.obolibrary.org/obo/ENVO_00000309>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000016> (material decumulation process)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.merriam-webster.com/dictionary/decumulation") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000016> "A process during which the mass of one or more materials, present within a given site, decreases.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000016> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000016> "material decumulation process"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000016> <http://purl.obolibrary.org/obo/ENVO_02500000>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000017> (glacial ice loss)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000017> "A process during which the mass of ice constituting a glacier decreases.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000017> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000017> "glacial ice loss"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000017> <http://purl.obolibrary.org/obo/ENVO_03000016>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000017> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00000133>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000017> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_03000004>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000018> (englacial stream)
```

Comment:
Great - https://github.com/EnvironmentOntology/envo/pull/481/commits/cf835a3d6ed1444a8e9e55132bbfd83f986ac28e fixed the build. 

Let's work on the axiomatisation a bit and then merge.